### PR TITLE
Estimator training

### PR DESCRIPTION
**Pull request by SamVanhoutte**

### CODE REVIEW

File: arcus/azureml/experimenting/resources/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+numpy
+pandas
+tqdm
+joblib
+matplotlib
+opencv-python
+seaborn
+argparse
+azureml-core
+arcus-azureml>=0.0.dev0
```
Comment (SriramAvatar) [inline arcus/azureml/experimenting/resources/requirements.txt:10]: I dont know what this means, just a double check if this is right?

File: tests/test_tuning_local.py [hunk 2]
```diff
@@ -8,13 +8,14 @@
 from sklearn import linear_model
 import pytest
 import arcus.azureml.environment.environment_factory as fac
+import arcus.azureml.experimenting.aml_trainer as tr
```
Comment (SriramAvatar) [inline tests/test_tuning_local.py:11]: Is this needed, I dont see an usage below.

File: arcus/azureml/experimenting/resources/requirements.txt (hunk 1 above)
Comment (SamVanhoutte) [reply arcus/azureml/experimenting/resources/requirements.txt:10]: it means that by default, the requirements file for an Estimator will download and take the latest version of our packages, including the dev/alpha versions.  I am considering to make this configurable over time

File: tests/test_tuning_local.py (hunk 2 above)
Comment (SamVanhoutte) [reply tests/test_tuning_local.py:11]: agreed - will remove in future PR

**Pull request merged**